docs/examples/tag/dismissable.js

```diff
@@ -0,0 +1,40 @@
+// @flow strict
+import { Fragment, type Element, type Node, useState } from 'react';
+import { Box, Button, Flex, Tag } from 'gestalt';
+
+export default function Example(): Node {
+  // eslint-disable-next-line no-use-before-define
+  const [tags, setTags] = useState<$ReadOnlyArray<Element<typeof Tag>>>([generateTag()]);
+
+  function generateTag() {
+    return (
+      <Tag
+        onRemove={() => {
+          setTags((currTags) => currTags.slice(0, currTags.length - 1));
+        }}
+        removeIconAccessibilityLabel="Remove tag"
+        text="Tag"
+      />
+    );
+  }
+
+  return (
+    <Box padding={2}>
+      <Flex direction="column" gap={3}>
+        <Button
+          onClick={() => {
+            setTags((currTags) => [...currTags, generateTag()]);
+          }}
+          text="Add tag"
+        />
+
+        <Flex alignItems="center" gap={2} height="100%" justifyContent="center" width="100%" wrap>
+          {tags.map((item, index) => (
+            // eslint-disable-next-line react/no-array-index-key
+            <Fragment key={index}>{item}</Fragment>
```

Comment:
Why not setting the key on the <Item> itself?

Reply:
The array is comprised of "rendered" Tags, so there isn't JSX to set the tag on. It's akin to the difference between something like
```js
[() => renderTag(), () => renderTag()]
```
and
```js
[renderTag(), renderTag()]
```